mappings/net/minecraft/client/gui/screen/TitleScreen.mapping

```diff
@@ -5,6 +5,7 @@ CLASS net/minecraft/class_442 net/minecraft/client/gui/screen/TitleScreen
 	FIELD field_17776 isMinceraft Z
 	FIELD field_18222 doBackgroundFade Z
 	FIELD field_19102 ACCESSIBILITY_ICON_TEXTURE Lnet/minecraft/class_2960;
+	FIELD field_23775 DEMO_LOGGER Lorg/apache/logging/log4j/Logger;
```

Comment:
If this is just a logger, then just rename it manually. enigma has some problems with these logger static final fields

Reply:
```suggestion
	FIELD field_23775 LOGGER Lorg/apache/logging/log4j/Logger;
```

Reply:
Enigma didn't let me use `LOGGER`, so I thought there would be a problem with it. It should now be `LOGGER`